src/Analysis/Ast/Test/AssignmentTests.cs

```diff
@@ -674,8 +674,7 @@ class D: ...
 if (x := 1234) == 1234:
     pass
 ";
-            var analysis = await GetAnalysisAsync(code);
-
+            var analysis = await GetAnalysisAsync(code, PythonVersions.Required_Python38X);
```

Comment:
You'll want to do this to the test below this one as well.

Reply:
Done!